index.html

```diff
@@ -615,6 +577,32 @@ <h2>
             <li>Resolve <var>p</var>.
             </li>
           </ol>
+          <p class="note" title="Payment Handler Install">
+            Calling <a>set()</a> for the first time is equivalent to installing
+            a payment handler for this site.
+          </p>
+          <p class="note" title="User Consent">
+            User consent for <a>set()</a> may be determined via any of the
+            following, for example:
+            <ul>
+              <li>Accepting or rejecting a permission prompt dialog at the time
+              of <a>set()</a> call</li>
+              <li>A user agent setting to always allow or deny <a>set()</a>
+              calls from specific origins</li>
+              <li>A user agent setting to always allow or deny all calls to
+              <a>set()</a></li>
+            </ul>
+          </p>
```

Comment:
@rsolomakhin, as I read the above note, it feels to me like it should be moved to the section on security and privacy considerations and rewritten something like this:

<p class='note'>
   This specification does not define how the user agent establishes user consent for <a>set()</a>. The user agent might prompt the user as part of the <a>set()</a> call, or allow the user manual configuration options for specific origins or all origins.
</p>

Reply:
Here's an alternative (in light of other parts of this thread):

<p>This specification does not define how the user agent establishes user consent when <a>set()</a> is first called. The user agent might prompt the user as a result of <a>set()</a>, or might not if consent has been established previously through manual configuration by the user or usage patterns.</p>

<p>User agents MAY reject a <a>set()</a> promise for security reasons (e.g., due to an invalid SSL certificate) and SHOULD notify the user when this happens.</p>

How does that sound?

Ian